core/src/execution/datafusion/expressions/cast.rs

```diff
@@ -274,16 +291,20 @@ impl Cast {
             (DataType::Utf8, DataType::Timestamp(_, _)) => {
                 Self::cast_string_to_timestamp(&array, to_type, self.eval_mode)?
             }
+            (DataType::Utf8, DataType::Date32)
+            | (DataType::Utf8, DataType::Date64) => {
+                Self::cast_string_to_date(&array, to_type, self.eval_mode)?
+            }
             (DataType::Int64, DataType::Int32)
             | (DataType::Int64, DataType::Int16)
             | (DataType::Int64, DataType::Int8)
             | (DataType::Int32, DataType::Int16)
             | (DataType::Int32, DataType::Int8)
             | (DataType::Int16, DataType::Int8)
-                if self.eval_mode != EvalMode::Try =>
-            {
-                Self::spark_cast_int_to_int(&array, self.eval_mode, from_type, to_type)?
-            }
+            if self.eval_mode != EvalMode::Try =>
```

Comment:
Seems like unnecessary re-formatting (multiple places).

Reply:
cleaned this up now with cargo fmt